transformations/p1_noun_transformation/requirements.txt

```diff
@@ -0,0 +1,5 @@
+setuptools
+beautifulsoup4==4.9.3
+nltk==3.6.2
+wptools @ git+git://github.com/pawan2411/wptools@30738f1
```

Comment:
This seems like a private repo without access. If this is not being used, you should remove it.